html/quote-generator.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20,500,0,-25" />
+    <link rel="stylesheet" href="../css/quote-generator.css">
+    <link rel="stylesheet" href="../css/normalize.css">
+    <title>Quote-generator</title>
+</head>
+
+<body>
+    <main class="page">
+        <div class="top-buttons">
+            <button class="top-button" id="all-authors-btn">all authors</button>
+
+            <button class="top-button" id="random-btn">
+                random <span class="material-symbols-outlined top-icon">
+                    autorenew
+                </span>
+            </button>
+        </div>
+
+        <div class="grid-container">
+            <div class="quote"></div>
+
+            <div class="quote-author-container">
+                <p class="quote-author"></p>
+                <p class="quote-genre"></p>
+            </div>
+        </div>
+
+        <div class="all-authors-container">
+            <h2 class="top-10-caption">Top 10 people with the most quotes:</h2>
+        </div>
+
+        <footer>
+            <p>Created by <strong><a class="link" href="https://github.com/Jijuzo">Jijuzo</a></strong> -
+                devChallenges.io
+            </p>
+        </footer>
+    </main>
+
+</body>
+<script src="../js/quote-generator.js"></script>
```

Comment:
As I remember script should be either part of head or body 🤔 (I can be wrong, but prove it 😄 )

So place it in the end of your body